Apps/Examples/Examples/SiwftUI Examples/AnnotationsExample.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+import SwiftUI
+@_spi(Experimental) import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct AnnotationsExample: View {
+    struct City {
+        var name: String
+        var location: CLLocationCoordinate2D
+    }
+
+    @State var camera = CameraState(center: .zero, zoom: 1)
+
+    var cities = [
```

Comment:
I guess this won't allow us to change annotations in runtime?
We can mark `cities` with @State but it will probably recreate the whole MapView each time annotations change, idk if `MapboxView` is fine with it.